server/server.js

```diff
@@ -1,16 +1,43 @@
 const express = require('express');
+const querystring = require('querystring');
+const bodyParser = require('body-parser');
+const dbConnection = require('../database/db_connection');
 const getAllCareers = require('../database/queries/get_all_careers');
+const addCareer = require('../database/queries/add_career');
 
 const app = express();
+app.use(bodyParser.json());
+app.use(
+  bodyParser.urlencoded({
+    extended: true
+  })
+);
 
 app.get('/api/careers', (request, response) => {
   getAllCareers((error, result) => {
     if (error) {
-      console.log(error);
+      response.send(
+        '<h1>Sorry, there was a problem with the data Please try again!</h1>'
+      );
     } else {
       response.send(result);
     }
   });
 });
 
+app.get('/add-career', (request, response) => {
+  response.sendFile('addCareer.html', { root: __dirname + '/../public/' });
+});
+
+app.post('/send-career', (request, response) => {
```

Comment:
Going back to the url thing above I agree with you above because the page is loaded to the user but this route is a post route and purely for adding to the db. Can we change the ones interacting with the db to have /api in them. Eg for this one /api/add-career?

Reply:
I disagree. It's not an api request, it's a post request that redirects to an express page.

For sure all actual api routes should be found under `/api`

I'd say rather than `send-career` this route should be `add-career` (having the same url doesn't matter if it's post vs get)